lib/javascript.d.ts

```diff
@@ -1265,7 +1268,7 @@ declare global {
 	/**
 	 * Executes a system command
 	 */
-	function exec(command: string, callback?: (err: Error | null | undefined, stdout: string, stderr: string) => void | Promise<void>): child_process.ChildProcess;
+	function exec(command: string, options?: Record<string, unknown>, callback?: (err: Error | null | undefined, stdout: string, stderr: string) => void | Promise<void>): child_process.ChildProcess;
```

Comment:
now it will not recognize the cb function if just two arguments passed. So either an overload or maybe we can just reexport the `child_process.exec` function completely, this will also give syntax completion and checking for the options parameter.

Reply:
How do I need todo it for the re-export?